app/appserver/static/js/handle_autoforward_old.js

```diff
@@ -0,0 +1,54 @@
+require([
+     "splunkjs/mvc",
+     ], function(mvc) {
+    var app_name = "TA-Databricks"
+ mvc.setFilter("dbquote", function(inputValue) {
+     return inputValue.replace(/"/g, '\\"').replace(/'/g, "\\'");
+   });
+   require.config({
+    paths: {
+        'jquery_databricks': '../app/' + app_name + '/js/jquery_databricks',
+        'underscore_utils': '../app/' +  app_name + '/js/underscore-min'
+        },
+    });
+ });
+
+
+
+
+require(
+    [
+        'jquery_databricks',
+        'underscore_utils',
+        'backbone',
+        "splunk.util"
```

Comment:
`splunkUtil` is not used

Reply:
@nfx Since this code was present in the repo already, we did not remove these unused imports. However we will update this PR with these changes.
 (File has been renamed here as handle_autoforward_old.js instead of  handle_autoforward.js),

Reply:
Removed splunkUtil